src/alerts/slack.coffee

```diff
@@ -4,9 +4,7 @@ request = require 'request'
 module.exports = class SlackAlert extends Alert
   sendToSlack: (message) ->
     options =
-      url: "#{@config.host}/services/hooks/incoming-webhook"
-      qs:
-        token: @config.token
+      url: "https://hooks.slack.com/services/#{config.token}"
```

Comment:
I think these are internal separations that shouldn't worry the user. I suggest we change the configuration all together to "webhook-url" and have the user paste the entire URL he receives when setting up a URL.